pkgs/desktops/gnome-3/core/evince/default.nix

```diff
@@ -66,13 +84,30 @@ stdenv.mkDerivation rec {
     libsecret
     librsvg
     adwaita-icon-theme
+    dbus
     gspell
-  ] ++ stdenv.lib.optional supportXPS libgxps;
+    texlive.bin.core # kpathsea for DVI support
+    t1lib
+  ] ++ stdenv.lib.optional supportXPS libgxps
+    ++ stdenv.lib.optionals supportMultimedia (with gst_all_1; [
+      gstreamer gst-plugins-base gst-plugins-good gst-plugins-bad gst-plugins-ugly gst-libav ]);
 
-  configureFlags = [
-    "--disable-nautilus" # Do not build nautilus plugin
-    "--enable-introspection"
-    (if supportXPS then "--enable-xps" else "--disable-xps")
+  mesonFlags = [
```

Comment:
Noticed that there's systemd user services.
Maybe we should enable those.

Reply:
Added nixos module for evince.

Reply:
Super :sparkle: